### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by airavata-elephant**

### CODE REVIEW

File: src/component/RegistrationPages/SignIn/RectangleLine/RectangleLine.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/SignIn/RectangleLine/RectangleLine.jsx:1]: delete empty row

File: src/component/RegistrationPages/SignIn/RectangleLine/RectangleLine.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import React from "react";
+import s from "./RectangleLine.module.css";
+import rectangleLine from "../../../../assets/img/socialNetwork/Rectangle-line.png";
+
+
+
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/SignIn/RectangleLine/RectangleLine.jsx:7]: delete empty row

File: src/component/RegistrationPages/SignIn/RectangleLine/RectangleLine.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+const RectangleLine = ( ) => {
+    return   <div className={s.contentSignIn__rectangleLine + ' rectangleLine'}>
+                 <img alt="rectangle-line" src={rectangleLine}/>
+                  <span className={s.rectangleLine__orBetweenLine}>or</span>
+                 <img alt="rectangle-line" src={rectangleLine}/>
+              </div>
+
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/SignIn/RectangleLine/RectangleLine.jsx:14]: delete empty row

File: src/component/RegistrationPages/SignIn/RectangleLine/RectangleLine.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import React from "react";
+import s from "./RectangleLine.module.css";
+import rectangleLine from "../../../../assets/img/socialNetwork/Rectangle-line.png";
+
+
+
+const RectangleLine = ( ) => {
+    return   <div className={s.contentSignIn__rectangleLine + ' rectangleLine'}>
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/SignIn/RectangleLine/RectangleLine.jsx:9]: you should put your jsx code in (). I mean write return with '()' - 

return (
  <div.......
);

File: src/component/RegistrationPages/SignIn/SignIn.module.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+.signInUser {
+  height: 14.39em;
+}
+
+.signInUser__fields {
+  height: 9em;
+}
+
+
+
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/SignIn/SignIn.module.css:23]: delete empty row

File: src/component/RegistrationPages/User.module.css [hunk 6]
```diff
@@ -1,43 +1,81 @@
...
-.user__input:focus:-moz-placeholder           {opacity:0;}/* Firefox 18- */
-.user__input:focus:-ms-input-placeholder      {opacity:0;}/* IE */
-
-.user__input{
+.user__input::-webkit-input-placeholder {
+  color: #8D8989;
+  opacity: 1;
+}
+
+/* webkit */
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/User.module.css:38]: this comment should be above the previous class

File: src/component/RegistrationPages/User.module.css [hunk 7]
```diff
@@ -1,43 +1,81 @@
...
+.user__input::-moz-placeholder {
+  color: #8D8989;
+  opacity: 1;
+}
+
+/* Firefox 19+ */
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/User.module.css:44]: this comment should be above the previous class

File: src/component/RegistrationPages/User.module.css [hunk 8]
```diff
@@ -1,43 +1,81 @@
...
+.user__input:-moz-placeholder {
+  color: #8D8989;
+  opacity: 1;
+}
+
+/* Firefox 18- */
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/User.module.css:50]: this comment should be above the previous class

File: src/component/RegistrationPages/User.module.css [hunk 9]
```diff
@@ -1,43 +1,81 @@
...
+.user__input:-ms-input-placeholder {
+  color: #8D8989;
+  opacity: 1;
+}
+
+/* IE */
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/User.module.css:56]: this comment should be above the previous class

File: src/component/RegistrationPages/User.module.css [hunk 10]
```diff
@@ -1,43 +1,81 @@
...
+.user__input:focus::-webkit-input-placeholder {
+  opacity: 0;
+}
+
+/* webkit */
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/User.module.css:61]: this comment should be above the previous class

File: src/component/RegistrationPages/User.module.css [hunk 11]
```diff
@@ -1,43 +1,81 @@
...
+.user__input:focus::-moz-placeholder {
+  opacity: 0;
+}
+
+/* Firefox 19+ */
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/User.module.css:66]: this comment should be above the previous class

File: src/component/RegistrationPages/User.module.css [hunk 12]
```diff
@@ -1,43 +1,81 @@
...
+.user__input:focus:-moz-placeholder {
+  opacity: 0;
+}
+
+/* Firefox 18- */
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/User.module.css:71]: this comment should be above the previous class

File: src/component/RegistrationPages/User.module.css [hunk 13]
```diff
@@ -1,43 +1,81 @@
...
+.user__input:focus:-ms-input-placeholder {
+  opacity: 0;
+}
+
+/* IE */
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/User.module.css:76]: this comment should be above the previous class

**Review by TarasOrlovskyi — CHANGES_REQUESTED** [review]

**Review by Vahtangius — DISMISSED** [review]

File: src/component/RegistrationPages/SignIn/RectangleLine/RectangleLine.jsx [hunk 14]
```diff
@@ -0,0 +1,14 @@
...
+import rectangleLine from "../../../../assets/img/socialNetwork/Rectangle-line.png";
+
+const RectangleLine = ( ) => {
+    return(
+      <div className={s.contentSignIn__rectangleLine + ' rectangleLine'}>
+                 <img alt="rectangle-line" src={rectangleLine}/>
+                  <span className={s.rectangleLine__orBetweenLine}>or</span>
+                 <img alt="rectangle-line" src={rectangleLine}/>
+              </div>
+    );
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/SignIn/RectangleLine/RectangleLine.jsx:12]: ctrl + alt + L

File: src/component/RegistrationPages/SignIn/RectangleLine/RectangleLine.jsx [hunk 15]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import s from "./RectangleLine.module.css";
+import rectangleLine from "../../../../assets/img/socialNetwork/Rectangle-line.png";
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/SignIn/RectangleLine/RectangleLine.jsx:3]: Rectangle-line.png => rectangle-line.png

File: src/component/RegistrationPages/SignIn/SignIn.jsx [hunk 16]
```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import RectangleLine from "./RectangleLine/RectangleLine.jsx";
+import SocialNetwork from "./SocialNetwork/SocialNetwork.jsx";
+import s from "./SignIn.module.css";
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/SignIn/SignIn.jsx:4]: change 's' to signInStyle and refactor in all file

File: src/component/RegistrationPages/SignIn/SocialNetwork/SocialNetwork.jsx [hunk 17]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import s from "./SocialNetwork.module.css";
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/SignIn/SocialNetwork/SocialNetwork.jsx:2]: change 's' to socialNetworkStyle and refactor all file

File: src/component/RegistrationPages/User.module.css [hunk 18]
```diff
@@ -19,25 +19,59 @@
...
-.user__input:focus:-moz-placeholder           {opacity:0;}/* Firefox 18- */
-.user__input:focus:-ms-input-placeholder      {opacity:0;}/* IE */
-
-.user__input{
+.user__input::-webkit-input-placeholder {
+  color: #8D8989;
+  opacity: 1;
+}
+
+/* webkit */
```
Comment (TarasOrlovskyi) [inline src/component/RegistrationPages/User.module.css:58]: did you change it?

**Review by TarasOrlovskyi — CHANGES_REQUESTED** [review]

**Review by Vahtangius — DISMISSED** [review]

File: src/component/RegistrationPages/User.module.css (hunk 18 above)
Comment (airavata-elephant) [reply src/component/RegistrationPages/User.module.css:58]: yes boss

**Review by TarasOrlovskyi — APPROVED** [review]

**Review by Vahtangius — APPROVED** [review]

**Pull request merged**